be/src/io/cache/block/cached_remote_file_reader.cpp

```diff
@@ -44,12 +44,13 @@ Status CachedRemoteFileReader::close() {
 
 std::pair<size_t, size_t> CachedRemoteFileReader::_align_size(size_t offset,
                                                               size_t read_size) const {
+    size_t segment_size = std::min(std::max(read_size, (size_t)4096), // 4k
+                                   (size_t)config::file_cache_max_file_segment_size);
```

Comment:
it is better to add a CONF_Validator(>=4096) for `file_cache_max_file_segment_size`

Reply:
thanks, done.